src/h5web/dataset-visualizer/vis/matrix/AnchorCell.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useContext } from 'react';
+import { FiAnchor } from 'react-icons/fi';
+import { StickyContext } from './StickyContext';
+import styles from './MatrixVis.module.css';
+
+interface Props {
+  style: React.CSSProperties;
+}
+
+function AnchorCell(props: Props): JSX.Element {
```

Comment:
Naming issue here: I find that the name of the component does not describe its functionality. Two solutions for me:
- Rename it `StickyToggler` or something similar
- Rename all `stickyIndices` to `anchorIndices` for consistency

Reply:
Finally kept like that to to keep the distinction between the button and the code handling the stickiness.